apps/protocol-frontend/src/components/DashboardShell.tsx

```diff
@@ -0,0 +1,157 @@
+import { useState, useEffect } from 'react';
+import { Box, Flex, Heading, Text, Link as ChakraLink } from '@chakra-ui/react';
+import * as _ from 'lodash';
+import { useUser } from '../contexts/UserContext';
+import PageHeading from './PageHeading';
+import { UIUser } from '@govrn/ui-types';
+import { ControlledSelect, Option } from '@govrn/protocol-ui';
+import ContributionsHeatMap from './ContributionsHeatMap';
+import { subWeeks } from 'date-fns';
+
+interface DashboardShellProps {
+  user: UIUser;
+}
+
+// fetching will need to happen on this page, will refetch after each filter update
+// passing in the User so that the name is available upon initial render
+
+const DashboardShell = ({ user }: DashboardShellProps) => {
+  const { allDaos, getUserContributionsCount } = useUser();
+  const [contributionsCount, setContributionsCount] = useState([]);
+  const [dateRange, setDateRange] = useState<number>(52);
+  const [selectedDaos, setSelectedDaos] = useState<any>([]);
```

Comment:
We should remove this any as it looks like it is a Set of {value, label}

Reply:
Good call!